pkg/mcs/scheduling/server/apis/v1/api.go

```diff
@@ -128,6 +131,9 @@ func (s *Service) RegisterSchedulersRouter() {
 	router := s.root.Group("schedulers")
 	router.GET("", getSchedulers)
 	router.GET("/diagnostic/:name", getDiagnosticResult)
+	router.GET("/config", getSchedulerConfig)
+	router.GET("/config/:name/list", getSchedulerConfigByName)
+	router.GET("/config/:name/roles", getSchedulerConfigByName) // compatibility for shuffle-region-scheduler
```

Comment:
Do we need these two?

Reply:
Yes, otherwise we'll modify shuffle region scheduler

Reply:
Or we may add a new interface for it

Reply:
unify to "/list" and use new api in pd-ctl